packages/world/src/systems/BuildSystem.sol

```diff
@@ -72,6 +75,16 @@ contract BuildSystem is System {
         BaseEntity._set(entityId, baseEntityId);
       }
     }
+    uint32 mass = ObjectTypeMetadata._getMass(objectTypeId);
+    Mass._setMass(baseEntityId, mass);
+    VoxelCoord memory shardCoord = coord.toForceFieldShardCoord();
+    ForceFieldMetadata._setTotalMassInside(
```

Comment:
mm does force field align with some other kind of shard? We might want to call it something more generic? not sure

Reply:
The force field shard is its own dimension to prevent overlaps of force fields. Renamed the variable from `shardCoord` to `forceFieldShardCoord`